Gemfile

```diff
@@ -15,7 +15,7 @@ gem "bourbon", "~> 4.2.0" # to keep in sync with getcalfresh
 gem "browser"
 gem "capybara"
 gem "delayed_job_active_record"
-gem "delayed_job_web", "~> 1.4.2"
+gem "delayed_job_web", ">= 1.4.3"
```

Comment:
Why the greater-than-equal-to operator rather than the pessimistic operator?

Reply:
It didn't seem like there was a reason to have a pessimistic operator—and I tried to remove that from a number of our gems that didn't need them, because I think we had accidentally let things get out of date—, but we want to make sure it doesn't regress to 1.4.1, which had the security vulnerability.